Translators/Incident/pgquery.py

```diff
@@ -0,0 +1,86 @@
+import os
+import sqlalchemy
+import logging
+
+db_config = {
+    # Pool size is the maximum number of permanent connections to keep.
+    "pool_size": 5,
+    # Temporarily exceeds the set pool_size if no connections are available.
+    "max_overflow": 2,
+    # Maximum number of seconds to wait when retrieving a
+    # new connection from the pool. After the specified amount of time, an
+    # exception will be thrown.
+    "pool_timeout": 30,  # 30 seconds
+    # 'pool_recycle' is the maximum number of seconds a connection can persist.
+    # Connections that live longer than the specified amount of time will be
+    # reestablished
+    "pool_recycle": 60  # 1 minutes
+}
+
+db = None
+
+def init_tcp_connection_engine(db_user, db_pass, db_name, db_hostname, db_port):
+    logging.info(f"Creating DB pool")
+    pool = sqlalchemy.create_engine(
+        # Equivalent URL:
+        # postgresql+pg8000://<db_user>:<db_pass>@<db_host>:<db_port>/<db_name>
+        sqlalchemy.engine.url.URL.create(
+            drivername="postgresql+pg8000",
+            username=db_user,  # e.g. "my-database-user"
+            password=db_pass,  # e.g. "my-database-password"
+            host=db_hostname,  # e.g. "127.0.0.1"
+            port=db_port,  # e.g. 5432
+            database=db_name  # e.g. "my-database-name"
+        ),
+        **db_config
+    )
+    #pool.dialect.description_encoding = None
+    logging.info("DB pool created!")
+    return pool
+
+def init_socket_connection_engine(db_user, db_pass, db_name, unix_query):
+  logging.info(f"Creating DB pool")
+  pool = sqlalchemy.create_engine(
+    # Equivalent URL:
+    # postgresql+pg8000://<db_user>:<db_pass>@/<db_name>?unix_sock=/cloudsql/<cloud_sql_instance_name>
+    sqlalchemy.engine.url.URL.create(
+      drivername="postgresql+pg8000",
+      username=db_user,  # e.g. "my-database-user"
+      password=db_pass,  # e.g. "my-database-password"
+      database=db_name,  # e.g. "my-database-name"
+      query=unix_query
+    ),
+    **db_config
+  )
+  logging.info("DB pool created!")
+  return pool
+
+
+def init_connection_engine():
+    db_user = os.environ["DB_USER"]
+    db_pass = os.environ["DB_PASS"]
+    db_name = os.environ["DB_NAME"]
+    if("INSTANCE_CONNECTION_NAME" in os.environ):
```

Comment:
Do we need to support the cloudsql instance now?

Reply:
Everything that we need access to should be in the cv-manager-postgres deployment. I checked the dev environment and it looks like we don't have cloudsql set up - are we planning to migrate to use cloudSQL? If so then yes we'll need to support it here.

Reply:
That was an old version with the cloudsql, I think we can pull it at this point.